src/app/state/rest/case.ts

```diff
@@ -217,3 +217,14 @@ export const useCaseDocuments = (caseId: Components.Schemas.Case["id"], options?
     isProtected: true
   })
 }
+
+export const useDocumentTypes = (options?: Options) => {
+  const handleError = useErrorHandler()
+  return useApiRequest<Components.Schemas.DocumentType[]>({
+    ...options,
+    url: makeApiUrl("document-types"),
```

Comment:
Dit endpoint geeft alle types, je moet de types hebben specifiek voor deze zaak(thema). Dat is deze '/api/v1/cases/{id}/document-types/'